index.js

```diff
@@ -237,14 +237,9 @@ class StaticComponentEnveloper {
   getAssets ({ chunks, compilation }) {
     const compilationHash = compilation.hash
     const publicPath = this.getPublicPath({ compilation })
-    const assets = {
-      js: [],
-      css: []
-    }
 
-    chunks.forEach((chunk) => {
-      // Prepend the public path to all chunk files
-      const chunkFiles = [].concat(chunk.files).map((chunkFile) => {
+    return chunks.reduce((assets, chunk) => {
+      const chunkFiles = chunk.files.map((chunkFile) => {
```

Comment:
I wonder if the [].concat was there to handle the case where chunks.files was undefined (or some such shenanigans?). Could you find any docs to give us some certainty around what we expect `chunks.files` to be - particularly in the case that there are indeed no chunks generated?

Reply:
That's a good point. The best thing that I found is this https://webpack.js.org/api/stats/#structure document, according to which the chunks are a `list`. However, they do not specify if the list could be `null` or `undefined`, so I'll try to guarantee that we are not trying to iterate over an `undefined`.